spec/static/main.js

```diff
@@ -184,3 +185,20 @@ app.on('ready', function () {
     }
   })
 })
+
+ipcMain.on('set-client-certificate-option', function (event, skip) {
+  app.once('select-client-certificate', function (event, webContents, url, list, callback) {
+    event.preventDefault()
+    if (skip) {
+      callback()
+    } else {
+      assert.equal(list.length, 1)
```

Comment:
Just curious, could this still be done in the render process?

Assertion failures in the main process cause the specs to immediately terminate on CI, where as in the render process they will continue to run and finish.